.github/workflows/pythonpublish.yml

```diff
@@ -225,3 +225,45 @@ jobs:
           file: ./plugins/flytekit-spark/Dockerfile
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+  build-and-push-flyin-images:
+    runs-on: ubuntu-latest
+    needs: deploy
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: "0"
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to GitHub Container Registry
+        if: ${{ github.event_name == 'release' }}
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: "${{ secrets.FLYTE_BOT_USERNAME }}"
+          password: "${{ secrets.FLYTE_BOT_PAT }}"
+      - name: Prepare flyin Image Names
+        id: flyin-names
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ghcr.io/${{ github.repository_owner }}/flytekit
+          tags: |
+            flyin-latest
+            flyin-${{ github.sha }}
+            flyin-${{ needs.deploy.outputs.version }}
+      - name: Push Flyin Image to GitHub Registry
+        uses: docker/build-push-action@v2
+        with:
+          context: "./plugins/flytekit-flyin/"
+          platforms: linux/arm64, linux/amd64
```

Comment:
this image only works on linux/amd64 for now, right?

Reply:
we should add a build-arg `arch` in the dockerfile, and pass it when building the image

Reply:
I can create a PR for all docker images in flyte-kit, all of them didn't do this